src/lein_tools_deps/plugin.clj

```diff
@@ -1,29 +1,35 @@
 (ns lein-tools-deps.plugin
   (:require [clojure.tools.deps.alpha :as deps]
             [clojure.tools.deps.alpha.reader :as reader]
+            [clojure.tools.deps.alpha.util.maven :as mvn]
+            (clojure [pprint :as pp]
+                     [reflect :as cr])
+            (clojure.tools.deps.alpha.extensions
+             [deps :as deps+]
+             [git :as git]
+             [local :as local]
+             [maven :as maven])
             [clojure.java.io :as io]
             [clojure.edn :as edn]
-            [leiningen.core.project :as p]
-            [leiningen.core.main :as lein]))
+            [clojure.tools.logging :as log]))
+  ;  [leiningen.core.project :as p]
+  ;  [leiningen.core.main :as lein]))
 
-#_(require 'clojure.tools.deps.alpha.extensions.deps) ;; broken ns in v0.3.260 should be fixed soon...
-(require 'clojure.tools.deps.alpha.extensions.git)
-(require 'clojure.tools.deps.alpha.extensions.local)
-(require 'clojure.tools.deps.alpha.extensions.maven)
+(def system (mvn/make-system))
+(log/debug "system" system)
 
+(def system-deps-file (io/file "/usr/local/lib/clojure/deps.edn"))
```

Comment:
I can't see this path on my system, I suspect this is a platform difference between the defaults on linux and mac.  Are you running on Linux?

Reply:
Yes I am using Linux. Do you know where the system deps.edn file is on Mac? I will put existence checks on it.

Reply:
Yeah it's here:

`/usr/local/Cellar/clojure/1.9.0.297/deps.edn`

The problem is that the tools.deps version number is in the path, we have an issue open for resolving this here https://github.com/RickMoynihan/lein-tools-deps/issues/6